pyronear/datasets/wildfire/fire_labeler.py

```diff
@@ -1,10 +1,13 @@
+#!usr/bin/python
+# -*- coding: utf-8 -*-
+
 import re
 from itertools import combinations
 
 import numpy as np
 
 
-class FireLabeler:
+class FireLabeler(object):
```

Comment:
All classes are new-style in python 3, so we no longer need to inherit from object.

Reply:
My bad, you are very correct